src/app/views/query-runner/request/auth/Auth.tsx

```diff
@@ -65,12 +65,16 @@ export function Auth(props: any) {
       <div>
         <div className={classes.accessTokenContainer}>
           <Label className={classes.accessTokenLabel}><FormattedMessage id='Access Token' /></Label>
-          <IconButton onClick={handleCopy} iconProps={copyIcon} title='Copy' ariaLabel='Copy' />
-          <IconButton iconProps={tokenDetailsIcon}
-            title={translateMessage('Get token details (Powered by jwt.ms)')}
-            ariaLabel={translateMessage('Get token details (Powered by jwt.ms)')}
-            href={`https://jwt.ms#access_token=${accessToken}`}
-            target='_blank' />
+          {(permissionModeType === PERMISSION_MODE_TYPE.User) &&
+            <IconButton onClick={handleCopy} iconProps={copyIcon} title='Copy' ariaLabel='Copy' />
+          }
+          {(permissionModeType === PERMISSION_MODE_TYPE.User) &&
+            <IconButton iconProps={tokenDetailsIcon}
+              title={translateMessage('Get token details (Powered by jwt.ms)')}
+              ariaLabel={translateMessage('Get token details (Powered by jwt.ms)')}
+              href={`https://jwt.ms#access_token=${accessToken}`}
+              target='_blank' />
+          }
```

Comment:
nitpick: could you not hide both components behind one flag? instead of one flag for each component?

Reply:
upvote

Reply:
ce8ad3526241d2adb7a2695a733ec38654b59cf9

Reply:
very nice lgtm